templates/beta-post.adoc

```diff
@@ -0,0 +1,186 @@
+---
+layout: post
+title: "TITLE"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GITHUB_USERNAME
+author_github: https://github.com/GITHUB_USERNAME
+seo-title: TITLE - makes sure it ends with - OpenLiberty.io
+seo-description: DESCRIPTION
+blog_description: DESCRIPTION
+---
+= TITLE
+AUTHOR_NAME <https://github.com/GITHUB_USERNAME>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+// // // // // // // //
+// Above:
+// Do not insert any blank lines between any of the lines above.
+// Do not remove or edit the variables on the lines beneath the author name.
+//
+// Replace TITLE with the blog post title eg: MicroProfile 3.3 is now available on Open Liberty 20.0.0.4
+// Replace GITHUB_USERNAME with your GitHub username eg: lauracowen
+// Replace DESCRIPTION with a short summary (~60 words) of the release (a more succinct version of the first paragraph of the post).
+// Replace AUTHOR_NAME with your name as you'd like it to be displayed, eg: Laura Cowen
+// // // // // // // //
+
+BETA_RELEASE_SUMMARY
+
+// // // // // // // //
+// Change the RELEASE_SUMMARY to a quick sentence intro. Make sure to keep the blank lines 
+// // // // // // // //
+We have two beta packages for link:{url-about}[Open Liberty]:
+
+* <<jakarta, Jakarta EE 9 Beta Features>>: a lightweight package that contains only the Jakarta EE 9 features.
```

Comment:
Can you change the text of these bullet points to match what we had in the latest beta? We'd switched them round so that you read about the all beta features package first, and modified the wording slightly to make sense in that context.

Reply:
Done